extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -35,7 +35,7 @@ private function getTitleMock( $type, $langCode, $artId, $artDbKey ) {
 	private function mockMonetizationModule() {
 		$name = 'MonetizationModuleHelper';
 		$mock = $this->getMock( $name );
-		$mock->method( 'getMonetizationUnits' )->willReturn( ['below_category' => 'testing'] );
+		$mock->method( 'getMonetizationUnits' )->willReturn( [ 'below_category' => 'testing' ] );
 		$mock->method( 'getWikiVertical' )->willReturn( 'other' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "getWikiVertical" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "other" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)